### DISCUSSION TITLE
Non blocking flow does not return anything

### DISCUSSION THREAD

**Discussion by micheljacquot34** [directus/directus#16821] [Archive]
Hi,

I'm trying to use a flow, for item.create, in non blocking mode.

There is nothing returned from the flow, so how can we link another action?

What I want to do, is to perform an action after the item has been created, as I need the ID (or key) of the newly created item.

Thank you for any tip.
Michel

**Comment by paescuj** [comment]
From the docs (https://docs.directus.io/configuration/flows/triggers.html#actions):

> The event's payload is copied into the $trigger of the flow.

**Reply by micheljacquot34** [reply]
Thank you, it works fine.
